app/src/main/java/org/mozilla/focus/activity/ScreenNavigator.java

```diff
@@ -100,6 +98,7 @@ public void popToHomeScreen(boolean animate) {
 
         this.mainMediator.clearAllFragment(false);
         this.mainMediator.showHomeScreen(animate, false);
+        this.browserMediator.setInForeground(false);
```

Comment:
popToHomeScreen() runs transaction without adding it to back stack, so we have to explicitly set browser fragment to background state here.